parsers/forth.c

```diff
@@ -0,0 +1,78 @@
+/*
+*   Copyright (c) 2023, Eric Forgeot
+*
+*   Based on work by Jon Strait
+*
+*   This source code is released for free distribution under the terms of the
+*   GNU General Public License version 2 or (at your opinion) any later version.
+*
+*   This module contains functions for generating tags for Forth files
```

Comment:
Any reference for the language?

Reply:
Forth is a very old language from the 70's. It seems to have some kind of revival now. I've put a reference URL in the newest commit